lib/matplotlib/container.py

```diff
@@ -107,6 +110,25 @@ def get_children(self):
 
 
 class BarContainer(Container):
+    """
+    Container for the artists of bar plots.
+
+    E.g. created in a :meth:`.Axes.bar` plot.
```

Comment:
This sentence no verb.

Something like
```
Container for the artists of bar plots (created by `~.Axes.bar`, etc.).
```
could fit as the first line of the docstring.

Same comment applies throughout PR.